In-house patch to add the 'solariszones' hypervisor to Nova's list of
known hypervisors.  This patch has not yet been submitted upstream.

--- nova-2015.1.2/nova/compute/hv_type.py.~1~	2015-10-13 07:52:44.000000000 -0700
+++ nova-2015.1.2/nova/compute/hv_type.py	2015-12-31 19:56:40.193951927 -0800
@@ -40,6 +40,7 @@ OPENVZ = "openvz"
 PARALLELS = "parallels"
 PHYP = "phyp"
 QEMU = "qemu"
+SOLARISZONES = "solariszones"
 TEST = "test"
 UML = "uml"
 VBOX = "vbox"
@@ -61,6 +62,7 @@ ALL = (
     PARALLELS,
     PHYP,
     QEMU,
+    SOLARISZONES,
     TEST,
     UML,
     VBOX,
